mask/eolearn/mask/cloud_mask.py

```diff
@@ -329,32 +312,40 @@ def _dilate_all(self, data):
 
         return data
 
-    def _ssim_stats(self, bands, is_data, win_avg, var, nt_rel, sigma):
+    def _ssim_stats(
+        self,
+        bands: np.ndarray,
+        is_data: np.ndarray,
+        local_avg: np.ndarray,
+        local_var: np.ndarray,
+        rel_tdx: int,
```

Comment:
what exactly is a `rel_tdx`?

Reply:
since `tdx` is a time index (see below), `rel_tdx` seemed like a better choice to represent the `tdx` relative to the `min_tdx`. At least compared to `rel_t_i`   ¯\\\_(ツ)\_/¯

Reply:
i never would've guessed it was relative to `min_tdx` :see_no_evil:
Lets leave it as `rel_<whatever>` and maybe we can fix it later.

Reply:
For now I actually removed it from this place and added it to the single function (where it's used) as a `local_t_idx`, which probably makes more sense, since everything `local_` is windows specific.

Anyway, we can discuss better names in in a live session, as mentioned below.